src/config/players.json

```diff
@@ -11,7 +11,10 @@
 				"%PROGRAMFILES(X86)%\\VideoLAN\\VLC"
 			],
 			"darwin": [
-				"/Applications/VLC.app/Contents/MacOS"
+				"/Applications/VLC.app/Contents/MacOS",
+				"~/Applications/VLC.app/Contents/MacOS",
```

Comment:
NodeJS doesn't resolve the tilde character to `$HOME` (it's just a shell thing). This needs to be changed to
`"$HOME/Applications/VLC.app/Contents/MacOS"`
so it can be resolved by the resolvePath wrapper method which doesn't support the tilde character either:
https://github.com/streamlink/streamlink-twitch-gui/blob/v1.8.1/src/app/utils/node/resolvePath.js#L19